set.mm

```diff
@@ -501775,6 +501788,75 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       WKWKWKWLCWRIXOXHCBWMEWNCWRFGHWOWPWQ $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Acyclic graphs
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c AcyclicGraph $.
+
+  $( Extend class notation with acyclic graphs. $)
+  cacycgr $a class AcyclicGraph $.
+
+  ${
+    $d g f p $.
+    $( Define the class of all acyclic graphs.  A graph is called _acyclic_ if
+       it has no (non-trivial) cycles.  (Contributed by BTernaryTau,
+       11-Oct-2023.) $)
+    df-acycgr $a |- AcyclicGraph =
+              { g | -. E. f E. p ( f ( Cycles ` g ) p /\ f =/= (/) ) } $.
+  $}
```

Comment:
I think it is OK (and conforms with other definitions of special graphs, see, for example, "regular graphs" ~df-rgr) to define "acylic sets" in general ( `g` need not be a "graph" in this definition). If actually graphs are concerned, then it should/can be added in the antecedent, e.g.,: `( ( G e. USGraph /\ G e. AcyclicGraph ) -> ...` ("If `G` is an acyclic simple graph, then ...").